packages/dev/docs/pages/react-spectrum/layout.mdx

```diff
@@ -0,0 +1,190 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+```tsx import
+import {Flex, Grid} from '@react-spectrum/layout';
+import {View} from '@react-spectrum/view';
+import {Picker, Item} from '@react-spectrum/picker';
+import Book from '@spectrum-icons/workflow/Book';
+import BulkEditUsers from '@spectrum-icons/workflow/BulkEditUsers';
+import Draw from '@spectrum-icons/workflow/Draw';
+import {Text} from '@react-spectrum/typography';
+```
+
+---
+category: Concepts
+---
+
+# Layout
+
+This page describes how to build application layouts with React Spectrum using Flex or Grid,
+as well as how to integrate with pre-defined Spectrum layouts using slots.
+
+## Introduction
+
+React Spectrum includes layout components to help build Spectrum compliant applications more easily.
+The [Flex](Flex.html) and [Grid](Grid.html) components are containers, which are responsible for
+the layout of their children. `Flex` follows the 
+[CSS flexbox](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Basic_Concepts_of_Flexbox)
+algorithm, while `Grid` implements
+[CSS grid](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout/Basic_Concepts_of_Grid_Layout).
+These components provide props with pre-defined Spectrum variables for sizing, spacing,
+and other layout options.
+
+In general, you should prefer using flex and grid over other CSS layout models. Spacing between components
+should be left to parent layout components rather than added as via margins to specific children. This helps
+ensure components are composable when reused in different places, and ensures that spacing is consistent.
+
+In addition to `Flex` and `Grid`, some React Spectrum components include pre-built layouts which you can 
+insert your content into via slots. This often happens automatically through the use of semantic elements,
```

Comment:
hmm I think there may have been a misunderstanding. I'm fine with calling them semantic elements as a concept. I think we were saying in the context of component categories and titles to avoid that technical term, but in this type of conceptual docs I think it makes sense. It's more describing how the components work, not what they're called in this case. 🤷

Reply:
any updates needed to @ktabors PR as a result?